package.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/ossf/package-feeds/blob/main/package.schema.json",
+    "title": "Package Schema Version 1.0",
+    "description": "The package representation as outputted by a ScheduledFeed",
+    "examples": [
+      "foobarpackage",
+      "1.0.0",
+      "2021-03-30T14:26:28",
+      "npm",
+      "1.0"
+    ],
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string",
+        "description": "The name of the package"
+      },
+      "version": {
+        "type": "string",
+        "description": "The package version, formatted respective to the given package type"
+      },
+      "created_date": {
+        "type": "string",
+        "description": "RFC 3339 timestamp representing the package creation date",
+        "format": "date-time"
+      },
+      "type": {
+        "type": "string",
+        "description": "The type of package"
```

Comment:
I am just wondering if this could be more descriptive to describe the origin of the package? Or maybe just an example could help.

Reply:
I think maintaining a list of valid values here is reasonable, given the mapping to FeedName in the repository

Reply:
There's an example of `npm` in the `examples` list, I do agree explicitly stating this maps to the `FeedName` is useful!